util/dvsim/utils.py

```diff
@@ -116,7 +116,7 @@ def _subst_wildcards(var, mdict, ignored, ignore_error, seen):
     and seen_err is true if we stopped early because of an ignored error.
 
     '''
-    wildcard_re = re.compile(r"{([A-Za-z0-9\_]+)}")
+    wildcard_re = re.compile(r"(?<!\$){([A-Za-z0-9\_]+)}")
```

Comment:
Hmm, ok, but this is a bit magical. I wonder whether it would be better to have some sort of explicit escaping (like Python's format function). So shell code that absolutely needed the braces (unusual) would end up looking like `${{foo}}`.